press/press/doctype/site/site.py

```diff
@@ -4647,3 +4671,31 @@ def create_subscription_for_trial_sites():
 		except Exception:
 			frappe.db.rollback()
 			log_error(title="Creating subscription for trial sites", site=trial_site)
+
+
+def archive_creation_failure_sites():
+	seven_days_ago = frappe.utils.add_days(frappe.utils.today(), -7)
```

Comment:
Can make it longer. 14 days perhaps.

The number 7 should be a capitalized constant variable at top of file like CREATION_FAILURE_GRACE_DAYS